.travis.yml

```diff
@@ -0,0 +1,115 @@
+language: cpp
+
+matrix:
+  include:
+    - os: linux
+      compiler: gcc
+      addons:
+        apt:
+          sources:
+            - ubuntu-toolchain-r-test
+          packages:
+            - gcc-6
+            - g++-6
+            - libopenmpi-dev
+            - openmpi-bin
+      before_cache:
+        - cd $TRAVIS_BUILD_DIR/../metis-4.0;
+          mv libmetis.a ..; rm -rf *; mv ../libmetis.a .
+
+install:
+   # Save Laghos top directory
+   - export CURR_DIR=$PWD
+
+   # Back out of the mfem directory to install the libraries
+   - cd ..
+
+   # hypre
+   - if [ ! -e hypre-2.10.0b/src/hypre/lib/libHYPRE.a ]; then
+        wget https://computation.llnl.gov/project/linear_solvers/download/hypre-2.10.0b.tar.gz --no-check-certificate;
+        rm -rf hypre-2.10.0b;
+        tar xvzf hypre-2.10.0b.tar.gz;
+        cd hypre-2.10.0b/src;
+        ./configure --disable-fortran --without-fei CC=mpicc CXX=mpic++;
+        make -j3;
+        cd ../..;
+     else
+         echo "Reusing cached hypre-2.10.0b/";
+     fi;
+
+   # METIS
+   - if [ ! -e metis-4.0/libmetis.a ]; then
+        wget http://glaros.dtc.umn.edu/gkhome/fetch/sw/metis/OLD/metis-4.0.3.tar.gz;
+        tar xvzf metis-4.0.3.tar.gz;
+        make -j3 -C metis-4.0.3/Lib CC="$CC" OPTFLAGS="-O2";
+        rm -rf metis-4.0;
+        mv metis-4.0.3 metis-4.0;
+     else
+        echo "Reusing cached metis-4.0/";
+     fi;
+
+   # MFEM
+   - if [ ! -e mfem/libmfem.a ]; then
+        rm -rf mfem;
+        git clone --depth 1 https://github.com/mfem/mfem.git;
+        cd mfem;
+        git checkout laghos-1.0;
```

Comment:
Should we make a new mfem tag?

Reply:
We will create an MFEM tag for the laghos-2.0 release, but I'm actually leaning towards testing MFEM master in the Laghosn Travis (we can make Laghis changes in the future that require changes in MFEM).

What do you think?

Reply:
I prefer master too.